Project-2/README.md

```diff
@@ -12,20 +12,45 @@ Tip #4: Work together as a team.
 1. Let V be a vocabulary of ALCI consisting of a role name "P". Interpret part_of as "x is a part of y". Using this role name, define the following formulas in this language:
 ```
   (a)  PP that says that x is a proper part of y
+  PP ≡ P ⊓ ¬P¯
+  
   (b)  iPP that says that y is a proper part of x
+  iPP ≡ PP¯ =>
+  iPP ≡ (P ⊓ ¬P¯)¯
+
+  iPP ≡ ¬P ⊓ P¯
+
   (c)  iP that says that x has y as part 
+  iP ≡ P¯
+  
   (d)  O that says that x overlaps y
+  O ≡ ∃P¯.(∃P)
+  O ≡ ∃iP.(∃P.⊤)
+  
   (e)  D that says that x and y are disjoint 
+  D ≡ ¬O
+  
 ```
 
 2. Use your axioms from question 1 as the basis of an ALCI T-Box. Supplement this T-box with whatever other axioms you like, as well as an A-box, so that you ultimately construct a knowledge base K = (T,A). Provide a _model_ of K. This may be graphical or symbolic or both. 
 
 3. Translate the following first-order logic axioms into ALCI: 
 ```
 (a) ∀x∃y∀z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+∃R.(∀R.(∀R¯))
+
 (b) ∃x∀y∃z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+∃R¯.(∃R.⊤) ⊓ ∃R.⊤
+
 (c) ∀y(R(x, y) → ∃x(R(y, x) ∧ ∀y(R(x, y) → A(y))))
+∀R.∃R.∀R.A
+Is this well formed?  Does it matter?
```

Comment:
It matters. Where do you think the grammar goes wrong?

Reply:
At first I thought there were too many parentheses at the end of the sentence, but that isn't the case.  This makes me think that it is well formed, and the answer is: ∀R.∃R.∀R.A